reana/reana_dev/run.py

```diff
@@ -98,6 +104,55 @@ def select_workflow_engines(workflow_engines):
     return list(output)
 
 
+def select_compute_backends(compute_backends):
+    """Return known compute backends names that REANA supports.
+
+    :param workflow_engines: A list of compute backends names such as 'slurmcern'.
+    :type components: list
+
+    :return: Unique compute backend names.
+    :rtype: list
+
+    """
+
+    def _has_cern_secrets():
+        """Check whether the test user has the correct CERN secrets?."""
+        output = subprocess.check_output(["reana-client", "secrets-list"]).decode(
+            "UTF-8"
+        )
+        required_cern_secrets = [
+            ".keytab",
```

Comment:
FWIW I'm not using `.keytab`, it would be nice to make this optional, or even better to check for the existence of a file name specified in `CERN_KEYTAB` variable, based on its value.

Reply:
Yes, the check can be better removed. In this way the command could serve to test situations such as missing CERN_USER, or missing keytab... So it is actually better if secrets are not checked at all; it would be the wrapper/caller, one layer above, that would take care of this.